google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -341,7 +341,7 @@ public void registerBreakpoint(
       BreakpointManager.getJavaBreakpoint(xIdeBreakpoint);
     if (!(cloudIdeBreakpoint instanceof CloudLineBreakpointType.CloudLineBreakpoint)) {
       LOG.error("breakpoint was not of the correct type to create on the cloud.  It was not a " +
-                "CloudLineBreakpoint");
+                "CloudLineBreakpoint"); //TODO akerekes: use class.getName() instead of hard coded name
```

Comment:
If the class is renamed, then I'd rather not rely on the refactoring tool to take care of it, regardless how much I trust it.